### ISSUE TITLE
suggested examples for contribution

### ISSUE THREAD

**Issue by dherman** [neon-bindings/examples-archive#1]
Here's a list of some single-serving examples that would make for great PRs. If you're interested in contributing one, just put "🔒 @yourusername" next to the item in the list. Once you have a PR open, add a link next to that, too.

  - [ ] [populate a JsBuffer](https://github.com/dherman/neon-examples/tree/master/populate-buffer) - 🔒 @dherman
  - [ ] same but for JsArrayBuffer
  - [ ] [nested scopes](https://github.com/dherman/neon-examples/tree/master/nested-scopes)
  - [ ] uses of `neon::vm::Lock`, e.g. mutating the contents of a binary buffer
  - [ ] doing simple parallel computation with Rayon
  - [ ] propagating exceptions with `?` syntax
  - [ ] defining classes
  - [ ] type testing and downcasting with `::check<T>()?`

**Comment by jugglingcats** [comment]
Is the idea that you are going to migrate examples from https://github.com/dherman/neon-examples?

**Comment by dherman** [comment]
@jugglingcats Yeah, and start linking to them from https://guides.neon-bindings.com
